lib/IRGen/GenValueWitness.cpp

```diff
@@ -885,10 +886,20 @@ void addStride(ConstantStructBuilder &B, const TypeInfo *TI, IRGenModule &IGM) {
 }
 } // end anonymous namespace
 
+bool irgen::layoutStringsEnabled(IRGenModule &IGM) {
+  auto moduleName = IGM.getSwiftModule()->getRealName();
+  if (IGM.Context.blockListConfig.hasBlockListAction(moduleName.str(), BlockListKeyKind::ModuleName, BlockListAction::ShouldUseLayoutStringValueWitnesses)) {
```

Comment:
Nit: `clang-format`

Reply:
You may also want to emit a diagnostic once somewhere indicating that the block list has taken effect to ensure that there is a record if when someone is confused about why it is disabled.